build.sbt

```diff
@@ -36,6 +35,11 @@ lazy val grpcService = (project in file("grpc-service"))
   .settings(
     javaAgents += "org.mortbay.jetty.alpn" % "jetty-alpn-agent" % alpnVersion % "runtime",
     dockerExposedPorts := Seq(8080),
+    dependencyOverrides ++= Seq(
```

Comment:
Do these need to be `overrides`? wouldn't adding them to the 'regular' dependencies do the trick?